### ISSUE TITLE
backbone's learning rate

### ISSUE THREAD

**Issue by Yzichen** [lucifer443/RecurrentBEV#2]
Does backbone's learning rate need to be adjusted when using pre-trained weights? For example, multiply by 0.2.

**Comment by lucifer443** [comment]
We didn't try this trick. Maybe the lr of backbone multiplied by 0.1 can get better performance.

**Comment by Yzichen** [comment]
May I ask if you modified it on solofusion? When training the model used for submission (VoVNet backbone), how many epochs did you train and did you use CBGS (number of iterations times 4.5)?

**Comment by lucifer443** [comment]
> May I ask if you modified it on solofusion? When training the model used for submission (VoVNet backbone), how many epochs did you train and did you use CBGS (number of iterations times 4.5)?

60 epochs without CBGS, other training settings are same with BEVDet
